mapping/process-level-implementation.csv

```diff
@@ -1,7 +1,7 @@
+Escalation of contract price,"Escalation to date of the price of materials, labour, equipment etc. due to fluctuations in inflation, currency etc.","Contracting process: For each escalation, publish an entry with a ``.date``, ``.description`` and ``.rationale`` in ``.summary.modifications`` with ``.type`` of 'value'. Provide the contract value before the variation in ``.oldContractValue`` and the contract value after the variation in ``.newContractValue``","OCDS: Monitor ``contracts.value`` for changes, copy rationale from ``contracts.amendments.rationale`` and date from ``release.date``",
+Variation to contract duration,Difference between original duration at contract award and the current projected duration in weeks.,"Contracting process: For each variation, publish an entry with a ``.date``, ``.description`` and ``.rationale`` in ``.summary.modifications`` with ``.type`` of 'duration'. Provide the contract period before the variation in ``.oldContractPeriod`` and the contract period after the variation in ``.newContractPeriod``","OCDS: Monitor ``contracts.period`` for changes, copy rationale from ``contracts.amendments.rationale`` and date from ``release.date``",
+Variation to contract scope,Any changes between original scope at contract award and the current scope,"Contracting process: For each variation, publish an entry with a ``.date``, ``.description`` and ``.rationale`` in ``.summary.modifications`` with ``.type`` of 'scope'","OCDS: Monitor ``contracts.description``, ``contracts.items`` and ``contracts.implementation.milestones`` for changes, copy rationale from ``contracts.amendments.rationale`` and date from ``release.date``",
+Reasons for price changes,"Summary of reasons for primary of changes (e.g. variations) that then lead to changes in contract price, major price fluctuations and accumulative increase or decrease in price.","Contracting process: For each variation, provide a ``.date``, ``.description`` and ``.rationale`` in ``.summary.modifications``.","OCDS: Copy from ``contracts.amendments.rationale`` and ``release.date``",
+Reasons for scope and duration changes,Summary of reasons for primary of changes (e.g. variations) that then lead to changes in the scope and duration.,"Contracting process: For each variation, provide a ``.date``, ``.description`` and ``.rationale`` in ``.summary.modifications``.","OCDS: Copy from ``contracts.amendments.rationale`` and date from ``release.date``",

```

Comment:
Use same construction for copying fields as suggested above.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/103/commits/b19fae52b688990a1f9bff8d500aba48ec3f9bb8